build.sbt

```diff
@@ -21,8 +21,6 @@ sonatypeProfileName := "org.virtuslab"
 
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
-ThisBuild / semanticdbEnabled := true
-ThisBuild / semanticdbVersion := scalafixSemanticdb.revision
```

Comment:
Pls comment out the scalafix plugin from plugins.sbt + add a TODO

Reply:
Don't remove it, just change the sbt behaviour based on the selected version!!

Reply:
Yeah, even better!

Reply:
I put a condition in `build.sbt` file to disable semanticdb if a scala version is `2.13.7` or `2.13.8`.